indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -415,7 +415,8 @@ def _check_access(self):
 
         # Check that template belongs to this event or a category that
         # is a parent
-        if self.template.owner != self.event and self.template.owner.id not in self.event.category_chain:
+        if (self.template.owner != self.event and
+                (not self.event.is_unlisted and self.template.owner.id not in self.event.category_chain)):
```

Comment:
shouldn't this be "unlisted OR ..."? it looks like right now you can use any template if you are in an unlisted event.

or even better: allow templates from the root category; those should always be available even for an unlisted event

Reply:
Removed the if and made it a bit clearer.